src/services/completions.ts

```diff
@@ -2665,6 +2670,10 @@ export function getCompletionEntriesFromSymbols(
         // expressions are value space (which includes the value namespaces)
         return !!(allFlags & SymbolFlags.Value);
     }
+
+    function symbolAppearsToBeTypeOnly(symbol: Symbol): boolean {
+        return !(symbol.flags & SymbolFlags.Value) && (!isInJSFile(symbol.declarations?.[0]?.getSourceFile()) || !!(symbol.flags & SymbolFlags.Type));

```

Comment:
1. It's a safe bet to treat anything in a JS file as possibly typed. Although I bet it's possible to do better, it's probably not worth the effort, though.
2. `isInJSFile` uses Node.flags to determine whether a node is in a JS file. And I thought the parser set that on everything in a JS file via `contextFlags` in `finishNode`. Does it work to use

```suggestion
        return !(symbol.flags & SymbolFlags.Value) && (!isInJSFile(symbol.declarations?.[0]) || !!(symbol.flags & SymbolFlags.Type));
```

or maybe `!symbol?.declarations.some(isInJSFile)`

Reply:
The removal of `?.getSourceFile()` works! But the different combinations of `.some(isInJSFile)` I tried each had test failures from type names being imported.